Sources/BraintreePayPal/BTPayPalLineItem.swift

```diff
@@ -27,16 +52,25 @@ import Foundation
     public let kind: BTPayPalLineItemKind
 
     /// Optional: Per-unit tax price of the item. Can include up to 2 decimal places. This value can't be negative or zero.
-    public let unitTaxAmount: String? = nil
+    public var unitTaxAmount: String? = nil
```

Comment:
This change is required to set values to these properties once the object is  created.